packpack

```diff
@@ -100,7 +100,7 @@ env | grep -E "PRODUCT|VERSION|RELEASE|TARBALL_|CHANGELOG_" > ${BUILDDIR}/env
 # Setup cache directory
 #
 CACHE_DIR="${XDG_CACHE_HOME:-$HOME/.cache}/packpack"
-mkdir -p "${CACHE_DIR}"
+mkdir -p "${CACHE_DIR}/ccache"
```

Comment:
AFAIK, this is not necessary, the directory will be created when and if ccache uses it (as long as the parent directory exists).